### ISSUE TITLE
Don't see any files in my home directory

### ISSUE THREAD

**Issue by Murmele** [flathub/org.flightgear.FlightGear#12]
When I try to add additional aircrafts, which are stored in my home folder, I'm not able to open them. There it looks like the home folder is empty.
Maybe
 - --filesystem=home

is missing in the finish-args?
I will create the flatpak package and try it out and report to here.

**Comment by Alexander-Wilms** [comment]
Are they added via a Open File dialog? Or do you just set the path?

**Comment by Murmele** [comment]
With a file dialog

Alexander Wilms <notifications@github.com> schrieb am Sa., 2. März 2019,
18:26:

**Comment by Alexander-Wilms** [comment]
I can try to reproduce this tomorrow. But if it uses the default Qt file dialog, it should use xdg-desktop-portal automatically.

**Comment by Murmele** [comment]
If I put the folder into .fgfs/Aircraft the folder will not be shown.

**Comment by TingPing** [comment]
It goes into `~/.var/app/org.flightgear.FlightGear/.fgfs` to be clear.

**Comment by Murmele** [comment]
ah I thought it is /home/user/.fgfs.
Yes with this path it works

**Comment by elehcim** [comment]
Also, no cache file is saved, between runs of the program no setting is kept.... I think there is some problem with filepath

**Comment by w-wagner** [comment]
Hi Guys, I have helped myself by opening the sandbox. It might not be the best way, but it helped:

Warning:
If you run debian and had an older fgfs version installed, the .fgfs folder in home dir is being upgraded. Running the old fgfs will not work anymore.

Edit: /var/lib/flatpak/app/org.flightgear.FlightGear/x86_64/stable/active/metadata

In block [Context] change 
```
filesystems=xdg-config/kdeglobals:ro;xdg-run/dconf;~/.config/dconf:ro;
```
into
```
filesystems=host;
```
After that all the issues I had worked:
- I could add a hangar
- the downloaded scenery is found after restart
- my settings are saved between sessions

Guess, that this setting is overwritten, when an update is released.

Maybe this workaround hints somebody how to fix it correctly.

**Comment by randyheydon** [comment]
Building on @w-wagner's workaround, I got it working by creating the .fgfs folder (i.e. `mkdir ~/.fgfs`) and appending it to the filesystems line:

    filesystems=xdg-config/kdeglobals:ro;xdg-run/dconf;~/.config/dconf:ro;~/.fgfs/;

The very next line has that folder marked as "persistent", but it looks like that's not working the way someone expected it to.

I'll also note that I installed FlightGear in per-user mode, so I found the metadata file under ~/.local/share/flatpac/app instead of /var/lib/flatpak/app.
